homeassistant/components/light/xiaomi_miio.py

```diff
@@ -18,16 +20,24 @@
 
 from homeassistant.const import (CONF_NAME, CONF_HOST, CONF_TOKEN, )
 from homeassistant.exceptions import PlatformNotReady
+from homeassistant.util import dt
 
 _LOGGER = logging.getLogger(__name__)
 
 DEFAULT_NAME = 'Xiaomi Philips Light'
 PLATFORM = 'xiaomi_miio'
 
+CONF_MODEL = 'model'
+
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Required(CONF_HOST): cv.string,
     vol.Required(CONF_TOKEN): vol.All(cv.string, vol.Length(min=32, max=32)),
     vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+    vol.Optional(CONF_MODEL, default=None): vol.In(
```

Comment:
Remove `default=None` and then you can remove `None` from the allowed values. You're already doing `config.get(CONF_MODEL)` so you won't run into trouble.